### PR TITLE
[HOTFIX] fix error table location path in HadoopFileExample

### PR DESCRIPTION
**Pull request by lamber-ken**
the table location path is lower case

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [x] Any interfaces changed? NO
 
 - [x] Any backward compatibility impacted? NO
 
 - [x] Document update required? NO

 - [x] Testing done YES
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required? NO
        - How it is tested? Please attach test report. NO
        - Is it a performance related change? Please attach the performance test report. NO
        - Any additional information to help reviewers in testing this change. It's a test example.
       
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/3325/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/11591/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3526/

File: examples/spark2/src/main/scala/org/apache/carbondata/examples/HadoopFileExample.scala [hunk 1]
```diff
@@ -61,7 +61,7 @@ object HadoopFileExample {
     CarbonInputFormat.setTableName(conf, "Hadoopfile_table")
 
 
-    val input = spark.sparkContext.newAPIHadoopFile(s"${storeLocation}/Hadoopfile_table",
+    val input = spark.sparkContext.newAPIHadoopFile(s"${storeLocation}/hadoopfile_table",
```
Comment (qiuchenjian) [inline examples/spark2/src/main/scala/org/apache/carbondata/examples/HadoopFileExample.scala]: why you modify only one,this file has serveral ```Hadoopfile_table```?

Comment (lamber-ken) [reply examples/spark2/src/main/scala/org/apache/carbondata/examples/HadoopFileExample.scala]: Thanks for your comment @qiuchenjian. I modify this one，then the example go through. I'll check again.

Comment (lamber-ken) [reply examples/spark2/src/main/scala/org/apache/carbondata/examples/HadoopFileExample.scala]: > why you modify only one,this file has serveral `Hadoopfile_table`?

I think it's no need to change other `Hadoopfile_table`. When use spark dataframe to save table, it'll use lower case table name as the location, so this pr fix the bug.

**Comment by lamber-ken** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/3368/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3571/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/11635/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.1/24/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.3.2, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/23/

**Pull request closed without merging**